### ISSUE TITLE
PERL WARNING: Use of uninitialized value in concatenation (.) or string at ./FHEM/10_EQ3BT.pm line 525.

### ISSUE THREAD

**Issue by djstkg** [mhop/fhem-mirror#37]
File        Rev   Last Change

10_EQ3BT.pm 17484 2018-10-07 18:54:14Z dominik

fhemweb.js                 17478 2018-10-07 16:45:18Z rudolfkoenig

**Comment by chrizz1001** [comment]
yeah, I am not so much into pearl... Anybody knows whats wrong here?

https://github.com/mhop/fhem-mirror/blob/29571e5cd9afe3798a1a230984b2b8f6a4487961/fhem/FHEM/10_EQ3BT.pm#L509-L525
